sigmastate/src/main/scala/sigmastate/eval/CostingDataContext.scala

```diff
@@ -461,6 +462,29 @@ object CHeader {
   * @see [[CostModel]] for detailed descriptions
   */
 class CCostModel extends CostModel {
+  import CCostModel._
+
+  def AccessBox: Int = AccessBoxCost
```

Comment:
ScalaDoc missed here and further

Reply:
added ScalaDocs in the CostModel trait and marked methods as `override`.
Some methods are not used, so marked them as `TODO refactor: ...`